DSC/README.md

```diff
@@ -78,47 +103,98 @@ $privateConfig = '{
   "MofFileName": "<mof-file-name>"
 }'
 
+$publicConfig = '{
+  "Mode": "Push"
+}'
+
 Set-AzureVMExtension -ExtensionName $extensionName -VM $vm -Publisher $publisher `
-  -Version $version -PrivateConfiguration $privateConfig | Update-AzureVM
+  -Version $version -PrivateConfiguration $privateConfig `
+  -PublicConfiguration $publicConfig | Update-AzureVM
+```
+
+#### 2.2.2.Resource Manager
+You can change to Azure Resource Manager mode by running:
+```powershell
+Switch-AzureMode -Name AzureResourceManager
+```
+You can deploy DSCForLinux Extension by running:
+```powershell
+$rgName = '<resource-group-name>'
+$vmName = '<vm-name>'
+$location = '<location>'
+
+$extensionName = 'DSCForLinux'
+$publisher = 'Microsoft.OSTCExtensions'
+$version = <version>
+
+# You need to change the content of the $privateConfig and $publicConfig 
+# according to different scenarios in section 3
+$privateConfig = '{
+  "StorageAccountName": "<storage-account-name>",
+  "StorageAccountKey": "<storage-account-key>",
+  "ContainerName": "<container-name>",
+  "MofFileName": "<mof-file-name>"
+}'
+
+$publicConfig = '{
+  "Mode": "Push"
+}'
+
+Set-AzureVMExtension -ResourceGroupName $rgName -VMName $vmName -Location $location `
+  -Name $extensionName -Publisher $publisher -ExtensionType $extensionName `
+  -TypeHandlerVersion $version -SettingString $publicConfig -ProtectedSettingString $privateConfig
 ```
 
-### 2.2. Apply a public MOF configuration file to the VM
+For more details about Set-AzureVMExtension syntax in ARM mode, please visit [Set-AzureVMExtension][Set-AzureVMExtension-ARM].
+
+### 2.3. Using [**ARM Template**][arm-template]
+
+For more details about ARM template, please visit [Authoring Azure Resource Manager templates](https://azure.microsoft.com/en-us/documentation/articles/resource-group-authoring-templates/).
```

Comment:
will update this section after the pull request (https://github.com/Azure/azure-quickstart-templates/pull/620)  is merged